static/js/app.js

```diff
@@ -11,14 +11,15 @@ require('nvd3');
 
 require('angular');
 require('angular-cookie');
+require('angular-pouchdb');
```

Comment:
@SCdF reordered the requires alphabetically. Only the `angular-translate-interpolation-messageformat` require is new.